Make ActiveStorage update migration compatible with install migration

etiennebarrie opened the pull request:
If you install ActiveStorage in rails 6.1, then use `rails app:update`, the service name migration handles that case but not the variant records one, which fails trying to re-create the table.

### Summary

I updated an app to Rails 6.1, but forgot to run `rails app:update` immediately. Then later I installed ActiveStorage, and finally came back to run the update task.
When I saw the update migrations being created in the app, I assumed they would fail but was pleasantly surprised that the service name passed by checking for the presence of the column. So I felt maybe we'd want to do the same to the other migration. We can also assume users will understand and just remove the migrations.

Repro steps:

```sh-session
$ rails _6.1.1_ new my_app && cd my_app
$ rails active_storage:install && rails db:migrate
$ rails app:update && rails db:migrate
# fails on CreateActiveStorageVariantRecords
```

### Other Information

It works perfectly in the regular case:

```sh-session
$ rails _6.0.3.4_ new my_app && cd my_app
$ rails active_storage:install && rails db:migrate
$ $EDITOR Gemfile && bundle update # update to 6.1
$ yes | rails app:update && rails db:migrate
```

rafaelfranca commented on activestorage/db/update_migrate/20191206030411_create_active_storage_variant_records.rb:
```diff
@@ -1,5 +1,7 @@
+  def up
+    return if table_exists?(:active_storage_variant_records)
+
     create_table :active_storage_variant_records do |t|
```
Let's change to 
```suggestion
    create_table :active_storage_variant_records, if_not_exists: true do |t|
```

etiennebarrie replied:
Oh right I had forgotten about this 👍 
Also works with as a reversible migration directly.

etiennebarrie commented:
This is still good to go!

etiennebarrie committed:
Make ActiveStorage update migration compatible with install migration

If you install active_storage in rails 6.1, then use rails app:update,
the service name migration handles that case but not the variant records
one, which fails trying to re-create the table.

Co-authored-by: Adrianna Chang <adrianna.chang@shopify.com>
